### PR TITLE
feat: add elevation.json

### PR DESCRIPTION
**Pull request by dwsilk**
#### Motivation

This tileset configures layer ordering for elevation data sources that will be provided as terrain-rgb and colour-ramp tiles.

#### Modification

Adds an initial elevation tileset configuration.

### CODE REVIEW

**Review by blacha — APPROVED** [review]

File: config/tileset/elevation.json [hunk 1]
```diff
@@ -0,0 +1,514 @@
+{
+  "id": "ts_elevation",
+  "name": "elevation",
+  "type": "raster",
+  "description": "elevation_description",
```
Comment (Wentao-Kuang) [inline config/tileset/elevation.json:5]: Might need a better description here.

File: config/tileset/elevation.json [hunk 2]
```diff
@@ -0,0 +1,514 @@
...
+  "title": "Elevation",
```
Comment (Wentao-Kuang) [inline config/tileset/elevation.json:6]:
```suggestion
  "title": "Elevation",
  "category": "Elevation",
```

File: config/tileset/elevation.json (hunk 1 above)
Comment (blacha) [reply config/tileset/elevation.json:5]: the aerial layer has it as "Aerial Imagery Basemap" so not exactly useful there either :P

Comment (blacha) [inline config/tileset/elevation.json:5]:
```suggestion
  "description": "Elevation Basemap",
```

**Review by blacha — APPROVED** [review]

**Review by Wentao-Kuang — APPROVED** [review]

**Pull request merged**